diff --git a/src/Driver/DrushRouter.php b/src/Driver/DrushRouter.php
index a555524..c316a2d 100644
--- a/src/Driver/DrushRouter.php
+++ b/src/Driver/DrushRouter.php
@@ -32,15 +32,15 @@ class DrushRouter {
   {
     try {
       $info = $target->getOptions();
-      $launchers = ['drush-launcher', 'drush.launcher', 'drush'];
+      $launchers = ['drush'];
       $binary = 'which ' . implode(' || which ', $launchers);
 
       $info = trim($target->exec('$(' . $binary . ') -r ' . $info['root'] . ' status --format=json'));
       $info = \Drutiny\Utility::jsonDecodeDirty($info, TRUE);
 
-      // Hack for Acquia Cloud to force use drush9 if drush version reported is v9.
+      // Hack for Lagoon to force use fixed path if drush version reported is v9.
       if (Comparator::greaterThanOrEqualTo($info['drush-version'], '9.0.0')) {
-        array_unshift($launchers, 'drush9');
+        array_unshift($launchers, '/app/vendor/drush/drush/drush');
         $binary = 'which ' . implode(' || which ', $launchers);
       }
 
